benchmarks/startup/meta.yaml

```diff
@@ -0,0 +1,10 @@
+name: startup-scenario
+run: "python run.py"
```

Comment:
```suggestion
run: "python -c \"import os; os.getenv('DDTRACE') == '1' and __import__('ddtrace')\""
```
?

Reply:
I opted against it in case we want to do more sophisticated logic in the `run.py` file also wanted to make it closer to a real-world python app in case `-c` has difference performance characteristics

Reply:
👍  Yep,  makes sense!